cli/cmd/help.go

```diff
@@ -155,8 +161,7 @@ Usage: {{ .Cmd }}
 {{end}}
 {{end -}}
 
-{{- if not .HasAvailableInheritedFlags}}` + util.Bold("\nEXTERNAL COMMANDS") + `
-%s
+{{- if not .HasAvailableInheritedFlags}} %s
```

Comment:
Please add an examples of the help output to the PR description. So, we can see it's format with external command and without.

Reply:
Added example to PR message